### PR TITLE
Translate the firmware flags when using fwupdmgr

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [X] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by hughsie** [comment]
Screenshot from 2022-01-24 17-03-28

File: src/fu-util-common.c [hunk 1]
```diff
@@ -1761,18 +1803,26 @@ fu_util_release_to_string(FwupdRelease *rel, guint idt)
...
+		if (title != NULL) {
+			g_autofree gchar *bullet = NULL;
+			bullet = g_strdup_printf("• %s", tmp2);
+			fu_common_string_append_kv(str, idt + 1, title, bullet);
+			title = NULL;
+		} else {
+			g_autofree gchar *bullet = NULL;
+			bullet = g_strdup_printf("• %s", tmp2);
+			fu_common_string_append_kv(str, idt + 1, "", bullet);
+		}
```
Comment (superm1) [inline src/fu-util-common.c:1823]: How about `title != NULL ? title : ""`?

**Review by superm1 — APPROVED** [review]
just one nit to remove some code lines, otherwise LGTM

File: src/fu-util-common.c (hunk 1 above)
Comment (hughsie) [reply src/fu-util-common.c]: Yes, and we can make it one line simpler too. Thanks.

**Pull request merged**